.CI/chatterino-installer.iss

```diff
@@ -56,7 +65,8 @@ SetupWindowTitle=Setup - %1 (Nightly)
 #endif
 
 [Tasks]
-Name: "vcredist"; Description: "Install the required Visual C++ 2015/2017/2019/2022 Redistributable";
+; Only show this option if the VCRT can be updated.
+Name: "vcredist"; Description: "Install the required Visual C++ 2022 Redistributable ({code:VCRTDescription})"; Check: NeedsNewVCRT();
```

Comment:
maybe instead of hardcoding `Visual C++ 2022 Redistributable` we could use VersionInfo.Product here?
it resolves to `Microsoft Visual C++ 2015-2022 Redistributable (x64) - 14.36.32532`, would save some maintenance in future

Reply:
Thank you! I was actually looking for something like this.

Reply:
Hmm, `VersionInfo.Product` doesn't seem to return anything for me (even though it's shown when stringifying the version info), so I used `VersionInfo.FileDescription`.

Reply:
both Product and FileDescription have same content on all versions i had on hand, so should be fine